modules/light-clients/08-wasm/types/genesis_test.go

```diff
@@ -1,17 +1,17 @@
 package types_test
 
 import (
-	"encoding/base64"
+	//"encoding/base64"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
-	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	//"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	//clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
 	ibctm "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint"
 	ibctesting "github.com/cosmos/ibc-go/v7/testing"
 )
 
-func (suite *TypesTestSuite) TestExportGenesisGrandpa() {
+/*func (suite *TypesTestSuite) TestExportGenesisGrandpa() {
```

Comment:
Q: We're waiting for the contract update for Grandpa, right? I'm guessing this is a temp commenting out for that eventuality.

Reply:
yes, we need updates for grandpa contract and hyperspace supporting the new wasm client changes. Then interchaintest support for any hyperspace cli/config changes (I'm pretty sure there are some). And then, finally, run interchaintest using compatible polkadot and parachain images. Once the e2e test is successful, we can harvest the test data for the wasm client's grandpa tests for the tests that require new test data.